tf/edgeml/trainer/bonsaiTrainer.py

```diff
@@ -411,26 +481,42 @@ def train(self, batchSize, totalEpochs, sess,
         # sigmaI has to be set to infinity to ensure
         # only a single path is used in inference
         bonsaiObjSigmaI = 1e9
-        print("\nMaximum Test accuracy at compressed" +
+        print("\nNon-Zero : " + str(self.getModelSize()[0]) + " Model Size: " +
+              str(float(self.getModelSize()[1]) / 1024.0) + " KB hasSparse: " +
+              str(self.getModelSize()[2]) + "\n", file=self.outFile)
+
+        if (self.bonsaiObj.isRegression == True):
+            maxTestAcc = np.mean(maxTestAcc)
+
+        print("Maximum Test accuracy at compressed" +
```

Comment:
Write a conditional for regression here as well. We don't want people to be confused.